doc/pravega-options.md

```diff
@@ -23,11 +23,13 @@ Here is an example,
 ...
 spec:
   pravega:
-    controllerJvmOptions: ["-XX:MaxDirectMemorySize=1g"]
-    segmentStoreJVMOptions: ["-XX:MaxDirectMemorySize=1g"]
+    controllerjvmOptions: ["-Xms1g", "-Xmx1g", "-XX:+ExitOnOutOfMemoryError", "-XX:+CrashOnOutOfMemoryError", "-XX:+HeapDumpOnOutOfMemoryError"]
+    segmentStoreJVMOptions: ["-Xms4g", "-Xmx4g", "-XX:MaxDirectMemorySize=11g", "-XX:+ExitOnOutOfMemoryError", "-XX:+CrashOnOutOfMemoryError", "-XX:+HeapDumpOnOutOfMemoryError"]
 ...
 ```
-We do not provide any JVM options as defaults within the operator code for the Controller or the Segmentstore. These options can be passed into the operator through the deployment manifest.
+We do not provide any JVM options as defaults within the operator code for the Controller or the Segmentstore. These options can be passed into the operator through the deployment manifest. Also we recommend setting both `-Xms` and `-Xmx` to the same value (as shown in the example above) so as to ensure that we save up the JVM growing memory.
+
+**NOTE:** For setting segementStore options **`-XX:MaxDirectMemorySize`**, **`-Xmx`** and **`pravegaservice.cache.size.max`** ,follow the guidelines provided in the [doc](https://github.com/pravega/pravega/blob/master/documentation/src/docs/admin-guide/segmentstore-memory.md)
```

Comment:
Some nits about this line:
- `segementStore` -> `Segment Store`
- ` ,follow` -> `, follow`
- `in the [doc]` -> `in this [document]`
- Missing final dot.